lib/plugins/create/templates/aws-java-maven/serverless.yaml

```diff
@@ -0,0 +1,51 @@
+# Welcome to Serverless!
+#
+# This file is the main config file for your service.
+# It's very minimal at this point and uses default values.
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    v1.docs.serverless.com
+#
+# Happy Coding!
+
+service: aws-java-maven
+provider:
+  name: aws
+  runtime: java8
+
+# you can overwrite defaults here
+#defaults:
+#  stage: dev
+#  region: us-east-1
+
+# you can add packaging information here
+package:
+#  include:
+#    - include-me.java
+#  exclude:
+#    - exclude-me.java
+  artifact: target/hello-0.0.1-SNAPSHOT.jar
```

Comment:
Is there a way we can tell maven to use a specific build path that doesn't change with the Version of the function? Otherwise it would have to be changed in both places all the time, which could lead to issues easily